docs/developer-guide/rules.md

```diff
@@ -189,7 +189,10 @@ Each rule must be accompanied by a README, fitting the following format:
 7.  Example patterns that are *not* considered violations (for each option value).
 8.  Optional options (if applicable).
 
-Look at the READMEs of other rules to glean more conventional patterns.
+Look at the READMEs of other rules to glean more conventional patterns. These includes:
```

Comment:
Grammar-wise, I believe it should be either _These include_ or _This includes_. These and those should be followed by a singular verb.